### PR TITLE
Add string representation to log level

### PR DESCRIPTION
**Pull request by leandromoreira**

### CODE REVIEW

File: log.go [hunk 1]
```diff
@@ -37,6 +37,11 @@ import "unsafe"
 
 type LogLevel int
 
+// https://github.com/FFmpeg/FFmpeg/blob/release/5.1/libavutil/log.c#L268
+func (l LogLevel) String() {
```
Comment (asticode) [inline log.go:41]: Could you move this function below constants?

**Comment by leandromoreira** [comment]
I see `char *get_level_str(int level)` is not exposed through log.h

**Comment by leandromoreira** [comment]
It seems it won't work since is not exposed :( 
```
log.cover.cgo2.c:(.text+0x25e): undefined reference to `get_level_str'
```

**Comment by leandromoreira** [comment]
I'll close it for now.

**Pull request closed without merging**